### ISSUE TITLE
Send and receive direct messages

### ISSUE THREAD

**Issue by amirpdlll** [LevPasha/Instagram-API-python#732]
Hello

Is there a way to read instagram direct messages and automatically reply them in this library ? 

thank you

**Comment by huguesdevimeux** [comment]
Yes, it is possible with `getV2inbox` to read the messages and `direct_message` to send some. 
It should work.

**Comment by LivC182** [comment]
I think with getV2inbox you can only get the last message from somebody not all unseen ones

**Comment by koon64** [comment]
The `getV2inbox` method now returns a `404`. Is there a new endpoint that Instagram has updated?

**Comment by LivC182** [comment]
Works fine for me. Post some code and I can debug it
